src/index.js

```diff
@@ -223,6 +228,7 @@ function createWebpackConfig(options) {
     output: {
       filename: outFile,
       path: outDir,
+      publicPath: options.outDir.replace(MATCHES_LEADING_DOT_SLASHES, ''),
```

Comment:
You should add this option only to `devServer`

https://webpack.js.org/configuration/dev-server/#devserver-publicpath-